src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.csproj

```diff
@@ -24,6 +31,10 @@
   <ItemGroup>
     <PackageReference Condition="$(BuildForRelease) == 'true'" Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="All" />
     <PackageReference Include="Azure.Core" Version="$(AzureCoreVersion)" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
```

Comment:
1.1.1 should move to our Versions.props file.

Reply:
GitHub source link package reference was moved to Versions.props file.